kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/KspOptions.kt

```diff
@@ -0,0 +1,344 @@
+/*
```

Comment:
Can we remove this file and KSPCommandLineProcessor.kt together? The latter is not used and the new entry point has moved away from KspOptions.

Reply:
Good catch, removed unused classes.